clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1825,12 +1878,13 @@ raw2trace_t::process_memref(raw2trace_thread_data_t *tdata, trace_entry_t **buf_
                             DR_PARAM_OUT bool *reached_end_of_memrefs,
                             DR_PARAM_OUT bool *interrupted)
 {
-    if (!append_memref(tdata, buf_in, instr, memref, write, reg_vals,
-                       reached_end_of_memrefs))
-        return false;
-    return handle_kernel_interrupt_and_markers(tdata, buf_in, cur_pc, cur_offs,
-                                               instr->length(), instrs_are_separate,
-                                               interrupted);
+    if (*interrupted) {
+        // Flush the memref entry.
```

Comment:
Explain that we do not try to indicate which memref might have caused a fault: we just omit them all, along with the instruction fetch.